reconcile/gabi_authorized_users.py

```diff
@@ -0,0 +1,96 @@
+import sys
+from datetime import datetime, date
+from typing import Iterable, Mapping, Optional
+
+from reconcile.utils.aggregated_list import RunnerException
+import reconcile.queries as queries
+import reconcile.openshift_base as ob
+from reconcile.utils.semver_helper import make_semver
+from reconcile.utils.openshift_resource import (OpenshiftResource as OR,
+                                                ResourceInventory as RI)
+from reconcile.utils.defer import defer
+
+QONTRACT_INTEGRATION = 'gabi-authorized-users'
+QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
+EXPIRATION_MAX = 90
+
+
+def construct_gabi_oc_resource(name: str, users: Iterable[str]) -> OR:
+    body = {
+        "apiVersion": "v1",
+        "kind": "ConfigMap",
+        "metadata": {
+            "name": name,
+            "annotations": {
+                "qontract.recycle": "true"
+            }
+        },
+        "data": {
+            "authorized-users.yaml": '\n'.join([u for u in users])
```

Comment:
You don't need the comprehension here, `join` takes an iterable:
```suggestion
            "authorized-users.yaml": '\n'.join(users)
```

Reply:
Also, will this work? Won't this produce a badly indented YAML? Shouldn't this be at least
```yaml
- user1
- user2
- user3
```
?

Reply:
Since JSON lists are compatible with YAML, maybe you'd prefer `json.dumps(users)` here?